src/Context/VideoContext.js

```diff
@@ -0,0 +1,13 @@
+import React, { createContext, useState } from 'react'
+
+export const VideoContext = createContext()
+
+export const VideoProvider = ({ children }) => {
+  const [videoLink, setVideoLink] = useState('https://www.youtube.com/watch?v=4C-gxOE0j7s')
+
+  return (
+    <VideoContext.Provider value={ [videoLink, setVideoLink ]}>
+      { children }
+    </VideoContext.Provider>
+  )
+}
```

Comment:
the more I look at this, I am really digging that this is in its own file. Of course, we'll see how it scales, but I like this way of doing it 👍🏼

Reply:
Agreed!! Earlier when we were meeting & you showed us this, you mentioned that you've sometimes seen line 3 as it's own file. Is this to break apart `createContext()` from the `Provider` piece? (No worries if we're still not clear on why some folks do it that way!)